javascript/javascript1/week2/Student manager/student.js

```diff
@@ -0,0 +1,52 @@
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+
+    if (studentName === 'Queen') {
+        class07Students.push(studentName);
+    }
+    else if (class07Students.includes(studentName)) {
+        console.log("Student " + studentName + " is already in the class");
+    }
+    else if (class07Students.length === 6) {
+        console.log("Cannot add more students to class 07.")
+    }
+    else if (studentName === "") {
+        console.log("Sorry cannot add a student without name!");
+    }
+    else {
+    class07Students.push(studentName);
+    }
+
+}
+
+function getNumberOfStudents() {
+    let classSize =  class07Students.length;
+    return classSize;
```

Comment:
You could simply return `class07Students.length` without assigning it to a variable! 😄